core/types/transaction.go

```diff
@@ -612,6 +613,16 @@ func (t *TransactionsByPriceAndNonce) Pop() {
 	heap.Pop(&t.heads)
 }
 
```

Comment:
Do you mind adding comments to these? Like how they are used

Reply:
Added.